.changelogs/deprecate-pricing-table.yml

```diff
@@ -0,0 +1,5 @@
+significance: minor
+type: deprecated
+links:
+  - "#197"
+entry: Deprecated course continue button, course progress and pricing table blocks.
```

Comment:
can you be speficic on the deprecated classes? you know why I'm asking this? because this is a dev deprecation, but the blocks will still be available, and users might get confused.
Also the course continue is not really deprecated here (it's js only), right? We just don't use it anymore, like "moved in the core plugin". Right?